diff --git a/rules/diff_test.bzl b/rules/diff_test.bzl
index 0f27f35..53d680b 100644
--- a/rules/diff_test.bzl
+++ b/rules/diff_test.bzl
@@ -171,9 +171,6 @@ def diff_test(name, file1, file2, failure_message = None, **kwargs):
         file1 = file1,
         file2 = file2,
         failure_message = failure_message,
-        is_windows = select({
-            "@bazel_tools//src/conditions:host_windows": True,
-            "//conditions:default": False,
-        }),
+        is_windows = False,
         **kwargs
     )
